mmseg/models/losses/lovasz_loss.py

```diff
@@ -302,3 +307,17 @@ def forward(self,
             avg_factor=avg_factor,
             **kwargs)
         return loss_cls
+
+    @property
+    def loss_name(self):
```

Comment:
Refer to https://github.com/open-mmlab/mmgeneration/blob/master/mmgen/models/gans/base_gan.py#L41 for understanding.

Reply:
Also https://github.com/open-mmlab/mmgeneration/blob/master/mmgen/models/gans/base_gan.py#L61

Reply:
https://github.com/open-mmlab/mmgeneration/blob/master/mmgen/models/gans/base_gan.py#L71

Reply:
https://github.com/open-mmlab/mmgeneration/blob/master/mmgen/models/gans/base_gan.py#L90

Reply:
https://github.com/open-mmlab/mmgeneration/blob/master/mmgen/models/gans/base_gan.py#L251